lib/Net/Async/Blockchain/ETH.pm

```diff
@@ -276,12 +284,12 @@ async sub _set_transaction_type {
     my $accounts_response = await $self->rpc_client->accounts();
     return undef unless $accounts_response;
 
-    my %accounts = map { $_ => 1 } $accounts_response->@*;
+    my %accounts = map { lc($_) => 1 } $accounts_response->@*;
```

Comment:
All this lowercasing seems fragile and easy to forget somewhere. Is it possible to do this somewhere higher up, or is the case significant in other uses?

Reply:
@turnstep this is a double-check the node will return the transaction in lowercase already, I'm adding this just for precaution.